align/schema/constraint.py

```diff
@@ -668,6 +611,7 @@ class Floorplan(UserConstraint):
     regions: List[List[str]]
     order: bool = False
     symmetrize: bool = False
+    _instance_attribute: str = "regions"
```

Comment:
@kkunal1408 , here is an example. In a future PR, we can refactor any such hard coding.

Reply:
okay